flows/routers/tests/tests.go

```diff
@@ -488,6 +489,23 @@ func HasDateGT(env utils.Environment, text types.XText, date types.XDateTime) ty
 	return testDate(env, text, date, isDateGTTest)
 }
 
+// HasTime tests whether `text` contains a time.
+//
+//   @(has_time("the time is 10:30")) -> true
+//   @(has_time("the time is 10:30 PM").match) -> 22:30:00.000000
+//   @(has_time("the time is 10:30:45").match) -> 10:30:45.000000
+//   @(has_time("there is no time here, just a number 5")) -> false
```

Comment:
I think this is correct - we're strict about dates now (you need year+month+day).. so seems time needs to be more than just a single number

Reply:
iso8601 says time can be simply `hh` so seems we should support just hour. (it also seems like a pretty common usecase really)

Reply:
I suppose my thinking here was that if you want to just ask for a number between 1 and 24 then we already have a rule for that... but yeah the common case is going to be people sending "2pm". I'll update our time parsing.

Should note that we don't currently support everything considered valid by ISO8601 - we don't support "hhmm" "hhmmss" or normalizing something like "24:00" to "00:00".

Reply:
https://github.com/nyaruka/goflow/issues/520